README.md

```diff
@@ -77,6 +77,25 @@ const myVariable = 99
 const os = process.platform === 'darwin' ? 'OSXy' /* c8 ignore next */ : 'Windowsy' 
 ```
 
+## Excluding Lines
+
+Sometimes you might find yourself wanting to exclude lines
+in your application (for example, perhaps you have your unit tests inside of a file
+you wish to check coverage of, but
+you don't want to include the tests in the report).
+
+To exclude lines, use the special comments `/* c8 exclude start */` and `/* c8 exclude stop */`.
```

Comment:
How is this different from [`/* c8 ignore start */` + `/* c8 ignore end */`](https://github.com/istanbuljs/v8-to-istanbul#ignoring-all-lines-until-told)? I guess the `ignore` marks lines as covered, but this `exclude` would not include them in the report at all - like `istanbul` instrumented code does?

Reply:
Just my opinion but I think this will be complex for users. This would introduce a second set of ignore hint syntax where:
- `ignore` would mark lines as covered even if they are not
- `exclude` would remove lines from coverage report - exactly as `istanbul` instrumented coverage does for `/* istanbul ignore ... */` hints

And this new `exclude` hint does not cover all the same cases as `ignore` does.

I'm not sure why exactly the `ignore` syntax has been implemented so that it works differently when compared to `istanbul` instrumented code. I would rather see it work like your `exclude` syntax here works so that it would completely remove the ignored parts from coverage report.

Reply:
I agree with you about the fact that ignore should work as exclude does.